apps/posters/models.py

```diff
@@ -41,6 +41,7 @@ class OrderMixin(models.Model):
         null=True
     )
     description = models.TextField(_(u"beskrivelse"), max_length=1000, blank=True, null=True)
+    comments = models.TextField(_(u"kommentar"), max_length=1000, blank=True, null=True)
```

Comment:
TextFields do not require a `max_length` attribute - the only thing this will do here is to limit the input text area to 1000 characters. If that's not the desired effect, just strip it from the model.